evm/src/executor/mod.rs

```diff
@@ -155,6 +166,16 @@ impl Executor {
         self
     }
 
+    /// Sets the fuzzer inspector of the executor.
+    pub fn set_fuzzer(
```

Comment:
I'm not sure I understand why this is here and also why `inspector_config_mut`/`inspector_config` exists - what's the thing you're trying to solve for here? Also for this function specifically wouldn't you be able to achieve the same thing with `inspector_config_mut().fuzzer = ...`?

Same comment as previously - it's a bit ambigious that this is called a fuzzer, but we also have a fuzzer outside of the executor